src/data/faq.json

```diff
@@ -2869,9 +2873,9 @@
                     "description": "Since version 2.2 of the Corona-Warn-App you are able to create a rapid test profile with your data such as first and last name, date of birth, address, phone number and e-mail address. Using a rapid test profile, you ensure that registration at a testing center is faster. The profile contains a QR code, which you can show at the testing center. However, you should still have your ID card ready."
                 },
                 {
-                    "term": "Recovery Certificate",
-                    "anchor": "recovery_certificate",
-                    "description": "The certificate of recovery is a <a href='#glossary_digital_certificate'>digital certificate</a> issued for a person who has demonstrably recovered from COVID-19 and proved immunity by a PCR test."
+                    "term": "Recovery certificate",

```

Comment:
Revert to the original capitalization "Recovery Certificate" using title case inline with other glossary entries.

Reply:
Fixed with [`a128219`](https://github.com/corona-warn-app/cwa-website/pull/2491/commits/a1282197b8ade5f9592a569015922557e715ceaf).